com.regnosys.rosetta/src/com/regnosys/rosetta/validation/RosettaValidator.xtend

```diff
@@ -96,6 +102,34 @@ class RosettaValidator extends AbstractRosettaValidator implements RosettaIssueC
 	@Inject RosettaGrammarAccess grammar
 	@Inject RosettaConfigExtension confExtensions
 	
+	static final Logger log = Logger.getLogger(RosettaValidator);
+	
+	protected override MethodWrapper createMethodWrapper(AbstractDeclarativeValidator instanceToUse, Method method) {
+		return new RosettaMethodWrapper(instanceToUse, method);
+	}
+	
+	protected static class RosettaMethodWrapper extends MethodWrapper {
+		protected new(AbstractDeclarativeValidator instance, Method m) {
+			super(instance, m)
+		}
+		
+		override void invoke(State state) {
+			try {
+				super.invoke(state);	
+			}
+			catch (Exception e) {
+				log.error(e);
```

Comment:
Add a little more info to the log.

Something like:

Validation {{method name}} failed due to unexpected validation exception.